Functions/Office365-Msgtrace/readme.md

```diff
@@ -0,0 +1,57 @@
+---
+page_type: sample
+products:
+- office-365
+- Sentinel
+languages:
+- powershellcore
+extensions:
+  contentType: samples
+  createdDate: 4/2/2020 3:00:56 AM
+description: "This sample can be used to process message traces from an Azure Function."
+---
+
+
+# Ingesting Office 365 Message Traces to Azure Sentinel
+
+Kudos to Stefan Simon! :)
+
+Implementing this script as part of an Azure Function will allow you to ingest Office 365 Message traces to Log Analytics.
+
+### Prerequisites
+
+You need to have an Azure Subscription, ability to create an Azure Function App. You need to have an account with permissions to run get-messagetrace in Office 365.
```

Comment:
Can you specify the exact permissions and in which platform (EXO/O365/Azure) they need to be assigned. The example would be a service account with only the permissions that are needed to run a MessageTrace call.